### PR TITLE
E0d/newrelic role

### PR DESCRIPTION
**Pull request by e0d**
Basic role for installing the Newrelic system agent.

### CODE REVIEW

File: playbooks/roles/newrelic/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+# vars are namespace with the module name.
+#
+newrelic_role_name: newrelic
+
+NEWRELIC_REPO: 'deb http://apt.newrelic.com/debian/ newrelic non-free'
+NEWRELIC_KEY_ID: '548C16BF'
+NEWRELIC_KEY_URL: 'https://download.newrelic.com/{{ NEWRELIC_KEY_ID }}.gpg'
+NEWRELIC_LICENSE_KEY: 'NEW-RELIC-KEY'
+
+NEWRELIC_APP_NAME: 'e0d-sandbox'
```
Comment (jarv) [inline playbooks/roles/newrelic/defaults/main.yml]: change the default?

File: playbooks/roles/newrelic/handlers/main.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+#
+#
+#
+# Handlers for role newrelic
+# 
+# Overview:
+# 
+#
+- name: notify me
+  debug: msg="stub handler"
```
Comment (jarv) [inline playbooks/roles/newrelic/handlers/main.yml]: i would just remove the handlers dir if there aren't any, up to you though

File: playbooks/roles/newrelic/meta/main.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+# Role includes for role newrelic
+# 
+# Example:
+#
+# dependencies:
+#   - {
+#   role: my_role 
+#   my_role_var0: "foo"
+#   my_role_var1: "bar"
+# }
```
Comment (jarv) [inline playbooks/roles/newrelic/meta/main.yml]: same comment for meta

**Comment by jarv** [comment]
:+1:

File: playbooks/edx-east/forum.yml [hunk 4]
```diff
@@ -5,6 +5,7 @@
   vars:
     enable_datadog: True
     enable_splunkforwarder: True
+    enable_newrelic: True
```
Comment (feanil) [inline playbooks/edx-east/forum.yml:8]: This should be disabled on the single instance sanboxes by default.  https://github.com/edx/configuration/blob/master/util/jenkins/ansible-provision.sh#L95-L96

**Comment by feanil** [comment]
How will upgrades of the newrelic agent work?

**Comment by e0d** [comment]
Per discussion, upgrades need to be handled in a more consistent way in general.  This role should be included in the edX base AMI play as it will be generally used.

**Comment by feanil** [comment]
looks good.

**Pull request merged**